source/overview/temporal.rst

```diff
@@ -1,6 +1,6 @@
 Time steps
 =================
-MESSAGE models the time horizon 1990 to 2110 in 5- and 10 year time steps where the first 5 periods (1990, 1995, 2000, 2005, 2010) are 5-year periods and the remaining 10 periods are 10-year periods (2020, 2030, 2040, 2050, 2060, 2070, 2080, 2090, 2100, 2110). The first four periods up to 2005 are fully calibrated, i.e. the model has no flexibility to change in these five periods. The 2010 period is partly calibrated so far, some recent trends are included in this time period, but some flexibility remains. In scenario applications the 2010 period is typically fixed to its baseline development so that future climate and energy policy cannot induce changes in the past. The reporting years are the final years of periods which implies that investments that lead to the capacities in the reporting year are the average annual investments over the entire period the reporting year belongs to.
+MESSAGE models the time horizon 1990 to 2110 in 5- and 10 year time steps where the first 5 periods (1990, 1995, 2000, 2005, 2010) are 5-year periods and the remaining 10 periods are 10-year periods (2020, 2030, 2040, 2050, 2060, 2070, 2080, 2090, 2100, 2110). The first five periods up to 2010 are fully calibrated, i.e. the model has no flexibility to change in these five periods. The 2020 period is partly calibrated so far, some recent trends are included in this time period, but some flexibility remains. In scenario applications the 2010 period is typically fixed to its baseline development so that future climate and energy policy cannot induce changes in the past. The reporting years are the final years of periods which implies that investments that lead to the capacities in the reporting year are the average annual investments over the entire period the reporting year belongs to. In some scenarios, the model has been run with 5-year modeling time steps by the middle of the Century (2015, 2020, 2025, 2030, 2035, 2040, 2045, 2050, 2055).
 
 MESSAGE can both operate perfect foresight over the entire time horizon, limited foresight (e.g., two or three periods into the future) or myopically, optimizing one period at a time (Keppo and Strubegger, 2010 :cite:`keppo_short_2010`). Most frequently MESSAGE is run with perfect foresight, but for specific applications such as delayed participation in a global climate regime without anticipation (Krey and Riahi, 2009 :cite:`krey_implications_2009`; O'Neill et al., 2010 :cite:`oneill_mitigation_2010`) limited foresight is used.
 
```

Comment:
Could be useful to add links to the respective parts of the MESSAGEix documentation where foresight is discussed.

Reply:
Thanks @volker-krey for this, I added a link to the mathematical formulation.